sw/legacy/test/memory_test.c

```diff
@@ -16,9 +16,11 @@
 #include "timer.h"
 
 #define TEST_DATA (0xDEADBEEF)
-#define TEST_SIZE ((224 * 1024) / 4)
+// Total ram size minus to space for .text and .rodata
+#define TEST_SIZE_BYTES (0x1F000 - 0x8000)
```

Comment:
This is a little more conservative than necessary; it can at least test another 4KiB, i.e. 1F000 - 7000. I reasoned that as follows - previously the code tested 224 and we've lost 128 so that leaves 224-128=96 that we can test.